Getting-Started-with-Android-Development.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with Android Development
+
+####Introduction
+Today [Android](https://www.android.com/) has become the leading Operating System being used in various devices, like mobile phone, tablets, watches, TV, cars and many more. It powers more than a billion devices all over the globe. It is based on the Linux kernel. Android has been the best selling OS on tablets since 2013, and on smartphones it is dominant by any metric.
+
+####Prerequisites
+For getting started with Android development, you should already know about a few things:
+* [Java](https://java.com/en/): Android development is mostly done in Java. It’s not a difficult language to learn, but knowing how it applies to Android development can be confusing without a firm foundation in the language. Android Java is not exactly standard Java, but learning standard Java will make it much easier for you to pick up Android development. For the specific topics in Java that would be helpful in this case, you can refer [this](http://www.bigknol.com/2016/01/10-prerequisites-for-learning-android.html) link.
+* [XML](http://www.w3schools.com/xml/): Knowing XML can be useful in a number of tech-related fields and Android is no exception. In particular, XML files make it easier to declare UI elements in the apps you create. Past experience is a big plus.
+* Perseverance: Lastly, learning the ins and outs of the Android development toolkit is going to take time. Learning how to create a quality app is going to take even more time. Don’t rush. Keep at it and focus your mind on the long term. If you don’t, you’ll be frustrated and want to give up.
+
+####Tools
+At first, make sure you have all the tools in your system. **Please note that the packages required to proceed with learning Android development require downloads of size more than 1 GB and the installation would take about 2 to 3 hours of your time.**
+The things you'll need to start building applications for Android are:
+* [Latest version of the Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html): [Here](https://www3.ntu.edu.sg/home/ehchua/programming/howto/JDK_Howto.html) is a guidfe to install Java 8 on your computer.
```

Comment:
Its good to link the Java installation instructions to our own wiki content, as mentioned in [Java Introduction](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Java-Introduction)